test/constants/blocks.js

```diff
@@ -28,6 +31,9 @@ const block = index => ({
   totalForged: `${30 + index}0000000`,
```

Comment:
Since the block function here might be used for 30 or 100 blocks, then maybe the constant passed here should be made a second argument for the function